NGRValidator/NGRValidator/Categories/NSString+NGRValidator.m

```diff
@@ -59,4 +105,43 @@ - (BOOL)evaluateAllowedCharacterSet:(NSCharacterSet *)allowedCharacterSet {
     return (range.location == NSNotFound);
 }
 
+- (BOOL)fulfillsLuhnAlgorithm {
```

Comment:
This method being private doesn't make it unsusceptible to collisions. It should be prefixed by `ngr_`, like others.